app/src/main/AndroidManifest.xml

```diff
@@ -9,8 +9,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".activities.VsJarvis"/>
```

Comment:
VsJarvis class is not needed as we can use the existing class GamePlayActivity.kt with Jarvis variant. Just pass the variant via intent and take the decision based on that variant.